### DISCUSSION TITLE
Protect a custom .ps1 file from deleting during update

### DISCUSSION THREAD

**Discussion by papadrams** [RainbowMiner/RainbowMiner#2034] [Q&A] [answered]
Hallo, 
As the title suggests, i have created a custom .ps1 file for claymore miner. I am running 2xRX480 and this miner is by far the fastest on Neoscrypt (1.98mh vs sgminerfancy 1.02mh vs nsgimner 150kh(wtf)).
After every update that .ps1 file gets deleted. Is there a way to protect it?
I red about minerconfigfiles.txt but that protects only in the bin folder.

**Comment by papadrams** [comment, marked as answer]
I just red #1718 and will try to have the .ps1 file in miners folder and minersoldversions folder.
I'll give you an update

**Comment by RainbowMiner** [comment]
Actually, if the ClaymoreNeoscrypt is really that much faster, I could re-add it. We had it in our library in the past.

**Reply by papadrams** [reply]
It would be good to be added back. Although it should be specific to older cards. It wouldnt work on newer amd cards. I know for a fact that it is faster on rx480, rx550 and rx570.

**Reply by RainbowMiner** [reply]
Ok, I will re-add it for the older AMD.

**Reply by papadrams** [reply]
Thanks!

**Reply by papadrams** [reply]
[AMD-ClaymoreNeoscrypt-GPU#00-GPU#01_NeoScrypt_HashRate.txt](https://github.com/RainbowMiner/RainbowMiner/files/8426827/AMD-ClaymoreNeoscrypt-GPU.00-GPU.01_NeoScrypt_HashRate.txt)
[AMD-Nsgminer-GPU#00-GPU#01_NeoScrypt_HashRate.txt](https://github.com/RainbowMiner/RainbowMiner/files/8426828/AMD-Nsgminer-GPU.00-GPU.01_NeoScrypt_HashRate.txt)
[AMD-SgminerFancyIX-GPU#00-GPU#01_NeoScryptXaya_HashRate.txt]
[AMD-SgminerFancyIX-GPU#00-GPU#01_NeoScrypt_HashRate.txt](https://github.com/RainbowMiner/RainbowMiner/files/8426831/AMD-SgminerFancyIX-GPU.00-GPU.01_NeoScrypt_HashRate.txt)

**Comment by papadrams** [comment]
Oh, and btw all 3 claymore miners are faster on my cards. (cryptonightr,neoscrypt and equihash) 
But it doesnt worth the effort to bring the other 2 back. They are not profitable at all

**Comment by RainbowMiner** [comment]
Could you please upload your implementation of the ClaymoreNeoscrypt miner module? I want to make sure, I have the parameters right.

**Comment by papadrams** [comment]
[2.txt](https://github.com/RainbowMiner/RainbowMiner/files/8426927/2.txt)
I entered the port manually (3333) cause i didnt know what i was supposed to do so look at that.

**Reply by papadrams** [reply]
In params i added my best result. -powlim 50 -a 2
In theory some may be better of with -a 1
